.github/workflows/test-address-sanitizer.yml

```
@@ -0,0 +1,59 @@
+#
+name: Test - Address Sanitizer
```

```suggestion
Build and Test - Ubuntu/gcc (Address Sanitizer)
```

This name should keep things grouped together better in the 'checks' section of PRs.